### DISCUSSION TITLE
TumblThree v2.9.3 released!

### DISCUSSION THREAD

**Discussion by thomas694** [TumblThreeApp/TumblThree#409] [Announcements]
＊*　'　*　.* 　* 🌟 ＊　*　'＊　*　*
＊　*　.*　'　 ✨ 🏮✨ 　＊　 ' 　＊
.　　　＊　 ✨ ☆☆☆✨　＊　　'　　.
　　*　　' ✨+:...💙...:✨　　*
'　　　　'✨☆ 🏮☆🏮☆✨　*　　＊　'　　
　＊　＊✨+:💚:+＠+💚:✨　　　'　　*
＊　.　✨☆☆ 📀☆📀☆☆ ✨ ＊　'　*　.
'　　. ✨+:..:++:..💚..+&:..:+✨
　*　✨☆🏮☆☆🏮☆☆🏮☆☆✨　＊　'　　*
'　. ✨+:...:+💙+:...:+💙§+:..:+✨
.　✨☆📀☆📀☆ ☆📀☆ 📀☆ ✨　　'　*
　✨💙+💙+:..:+@+:..:+💙+💙✨
　.　　*　.　　. 🟫 　 '　' ＊　　　　　*
*　　*　.　*　* 🟫　.　　*　.　　'
　 ,·´ ¸,·´`) * * 💝🎁🎁💝　*
(¸＊＊ ℳ𝒆𝓻𝓻𝔂 𝓒𝓱𝓻𝓲𝓼𝓽𝓶𝓪𝓼 ＊＊
　✦ ✦ ꝸ 𝑯͋𝒂͓𝒑̽𝒑̟𝒚 𝑵͋𝒆͓𝒘 𝒀̽𝒆̟𝒂͋𝒓 ✦ ✦

### TumblThree 2.9.3

- Bug fixes
  - Hidden crawler stops working because TumblThree gets logged out (one fix) (issue #404)
  - Blog crawler seems to rescan whole blog (issue #404)
  - Downloading liked posts from a date range doesn't work (issue #401)
  - Problem adding hidden blogs (issue #162)
  - Column widths are always reset to minimum value (issue #152)
  - Window dimensions are lost on startup (issue #406)
  - Occasional error parsing post
  - Exception when WebView2 crashes internally

<hr /><em>This discussion was created from the release <a href='https://github.com/TumblThreeApp/TumblThree/releases/tag/v2.9.3'>TumblThree-v2.9.3</a>.</em>

**Comment by vozmor** [comment]
Merry Christmas to you, too, and thank you very much for your efforts and maintaining this great app!
